invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/member_view/MembersResultItem.js

```diff
@@ -55,6 +58,18 @@ const dropdownVisibilityOptionsGenerator = (value) => {
 };
 
 class MemberViewResultItem extends Component {
+  static contextType = ModalContext;
+
+  constructor(props) {
+    super(props);
+
+    const {
+      community: { uuid },
+    } = props;
+
+    this.membersApi = new CommunityMembersApi(uuid);
```

Comment:
Is this created in each result item?
If yes, it should be created once, at the app level on `init`, probably as a singleton, and then use it around.